Library/Formula/doctl.rb

```diff
@@ -3,8 +3,8 @@
 class Doctl < Formula
   desc "Command-line tool for DigitalOcean"
   homepage "https://github.com/digitalocean/doctl"
-  url "https://github.com/digitalocean/doctl/archive/0.0.16.tar.gz"
-  sha256 "4f4805f36fd0d437331c25a183471419d10a680721f7f5a890b4109319d605ed"
+  url "https://github.com/phillbaker/doctl/archive/0.0.17.tar.gz"
```

Comment:
Whoops, thanks for catching that. I was testing and pushed a tag of existing code to my fork. I'll bug them to see if they can do the same.